src/main/java/mate/academy/spring/model/dto/request/ShoppingCartRequestDto.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.spring.model.dto.request;
+
+import java.util.List;
+
+public class ShoppingCartRequestDto {
+    private List<Long> ticketIds;
```

Comment:
this request dto redundant
see https://mate-academy.github.io/jv-program-common-mistakes/java-spring/rest-practice/jv-spring-rest-practice_checklist